dev/ProjectReunion_DLL/dllmain.cpp

```diff
@@ -0,0 +1,19 @@
+// dllmain.cpp : Defines the entry point for the DLL application.
+#include "pch.h"
+
+BOOL APIENTRY DllMain( HMODULE hModule,
```

Comment:
I would remove this entirely. This is just inviting someone to do something fun inside DllMain. 😉